web/packages/teleport/src/components/ServersideSearchPanel/ServersideSearchPanel.tsx

```diff
@@ -55,6 +54,7 @@ export function ServersideSearchPanel({
       onSubmit={onSubmitSearch}
       borderTopLeftRadius={3}
       borderTopRightRadius={3}
+      style={disabled ? { pointerEvents: 'none', opacity: '0.5' } : {}}
```

Comment:
There was no way to disable before this PR?

Reply:
It doesn't appear so.